Notification Content/NotificationViewController.swift

```diff
@@ -45,10 +45,14 @@ class NotificationViewController: UIViewController, UNNotificationContentExtensi
         case .connectionRequest:
             guard let connectionId = notification.connectionId else { return }
             self.view.addSubview(self.connectionRequestView)
-            Connection.localThenNetworkQuery(for: connectionId)
-                .mainSink { connection in
-                    self.connectionRequestView.configure(with: connection)
-                }.store(in: &self.cancellables)
+
+            Task {
+                guard let connection = try? await Connection.localThenNetworkQuery(for: connectionId) else {
+                    return
+                }
+
+                self.connectionRequestView.configure(with: connection)
+            }
```

Comment:
Query calls are also async. In this example we're not handling the error. We probably should in the future but I'm just refactoring the existing code for now.